internal/provider/resource_model.go

```diff
@@ -79,11 +79,10 @@ func resourceModelCreate(ctx context.Context, d *schema.ResourceData, meta inter
 	var diags diag.Diagnostics
 
 	name := d.Get("name").(string)
-	controller := d.Get("controller").(string)
```

Comment:
Does this mean we no longer need to specify this for a `juju_model` resource? 

If that's the case it should be removed from the resource schema (and example docs).

Reply:
This will be addressed in a separate PR